### DISCUSSION TITLE
Panlee ZX7D00CE01S display shift

### DISCUSSION THREAD

**Discussion by Materie62** [moononournation/Arduino_GFX#586] [Q&A]
Hi,
using the Arduino_GFX library I created a program that displays a real life ADC signal. What I noticed is that the complete display moves from right to left in small (random ?) steps. It's almost as if lines are drawn relatively with a small position error. However I'm definitely using absolute coordinates. And please note that the *complete* screen moves, including a small BMP picture.
The effect seems to get worse when using the touch capabilities of the screen. 

Did anyone see this effect? And what to do?
Greetings.

**Comment by moononournation** [comment]
yes, it is common symptom for large resolution RGB display

**Reply by Materie62** [reply]
This doesn't sound promising. What would be the correct approach to solve this issue? And do you know why this is happening? Perhaps writing too fast, too many times or....

**Reply by moononournation** [reply]
your program loop holded too long, RGB display cannot update in time

**Reply by moononournation** [reply]
you may consider other display interfaces, I tried QSPI can be much better
